app/routers/api/auth.py

```
@@ -244,3 +255,354 @@ def generate_api_token(
+        key="oidc_state",
+        value=state,
+        httponly=True,
+        secure=request.url.scheme == "https" or request.headers.get("x-forwarded-proto") == "https",
+        samesite="lax",
+        max_age=300,  # 5 minutes
+    )
+    return response
+
+
+@router.get("/oidc/callback", response_class=HTMLResponse)
+async def oidc_callback(
```

Ruff (0.15.20)

[warning] Too many branches (29 > 12)

(PLR0912)